config/environments/production.rb

```diff
@@ -20,6 +20,9 @@
   # Generate digests for assets URLs
   config.assets.digest = true
 
+  # ensure we can use ES6 in production
+  # config.assets.js_compressor = :uglifier
```

Comment:
I guess this was added thinking we needed it, but we don't @tansaku?

Reply:
yes, it's an optional setting we can use to lock to ES6 if we want, but it turned out not to be helpful - but I thought I'd leave it there so it's clear it's an option in future